include/revng/Model/DisassemblyConfiguration.h

```diff
@@ -30,6 +34,10 @@ class model::DisassemblyConfiguration
   : public model::generated::DisassemblyConfiguration {
 public:
   using generated::DisassemblyConfiguration::DisassemblyConfiguration;
+  DisassemblyConfiguration() : generated::DisassemblyConfiguration() {
+    // TODO: introduce a better way of specifying default values.
+    UseIntelSyntax() = true;
+  }
```

Comment:
Mmmmh, I think this will create problems with all the other wrappers (Python and TypeScript).

Can we have `UseATTSyntax`? I know it's not ideal, but...

Reply:
Done, `UseIntelSyntax` -> `UseATTSyntax`. As for others, I added
```cpp
if (SelectedValue == SomeEnum::Invalid)
  SelectedValue = SomeEnum::Default;
```

which should do for now.